### PR TITLE
Feature/carousel

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: .carousel test.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.carousel {
```
Comment (Veryyapeee) [inline .carousel test.html]: I guess this is some kind of the test file that you should get rid of :D

File: src/components/TitlesCarousel/TitlesCarousel.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    display: flex;
+    .carousel__prev, .carousel__next {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    outline: 0;
+    cursor: pointer;
+    position: absolute;
+    }
+    .carousel__prev {
```
Comment (Veryyapeee) [inline src/components/TitlesCarousel/TitlesCarousel.scss]: I guess you can move these styles to the .carousel{} and use the &__prev syntax

Comment (Mariusz-web) [reply src/components/TitlesCarousel/TitlesCarousel.scss]: prev and next are in a wrapper, not a carousel so I can change class

File: .carousel test.html (hunk 1 above)
Comment (Mariusz-web) [reply .carousel test.html]: How? Because i dont know how to delete this

File: src/components/TitlesCarousel/TitlesCarousel.scss (hunk 2 above)
Comment (Mariusz-web) [reply src/components/TitlesCarousel/TitlesCarousel.scss]: done

File: src/components/TitlesCarousel/TitlesCarousel.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    &__content {
+        display: grid;
+        grid-gap: 1em;
+        grid-auto-flow: column;
+        margin: auto;
+        box-sizing: border-box;
+    }
+
+    &__item {
```
Comment (Veryyapeee) [inline src/components/TitlesCarousel/TitlesCarousel.scss]: I think that we don't need these styles anymore

File: src/screens/Home.js [hunk 4]
```diff
@@ -1,17 +1,18 @@
 import { render } from '../shared/dom.js';
-import { MovieTile } from '../components';
+import { MovieCarousel } from '../components';
 import ApiProvider from '../providers/ApiProvider.js';
+import { MovieTile } from '../components';
```
Comment (Veryyapeee) [inline src/screens/Home.js]: This is unused, remove this

File: src/screens/Home.js [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
 import { Star } from '../components/index.js';
```
Comment (Veryyapeee) [inline src/screens/Home.js]: This is unused too, remove that pls

File: src/screens/Home.js [hunk 6]
```diff
@@ -20,5 +21,5 @@ export const HomeScreen = async ({ renderOn, options }) => {
     const movie = await ApiProvider.getTitleDetails(my_movie.id);
```
Comment (Veryyapeee) [inline src/screens/Home.js]: movie, and movie_tile is unused, remove that pls

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/screens/Home.js [hunk 7]
```diff
@@ -1,24 +1,22 @@
 import { render } from '../shared/dom.js';
-import { MovieTile } from '../components';
+import { MovieCarousel } from '../components';
 import ApiProvider from '../providers/ApiProvider.js';
 import { Star } from '../components/index.js';
```
Comment (Veryyapeee) [inline src/screens/Home.js]: The Start import which is unused is still here :P

**Review by Veryyapeee — APPROVED** [review]

**Review by T0ffik — APPROVED** [review]

**Pull request merged**